DIRECTORY-STRUCTURE.md

```diff
@@ -37,6 +37,7 @@ This may not be comprehensive for every file, but should be considered a living
 │   ├── os [Bottlerocket's documentation]
 │   │   ├── 1.12.x [versioned content, each new minor version gets a new directrory]
 │   │   │   ├── introduction
+│   │   │   ├── packages [which packages (and which versions) are included in this release]
```

Comment:
I _think_ this works but how will we handle different packages at different patch versions?

Reply:
Would the following work?

A subdirectory for each patch version under the `packages` directory:

e.g.: `packages/1.12.0`, `packages/1.12.1`, `packages/1.12.2`, etc?

Alternatively, would listing a separate package-version table for each patch version (on the same page) work?

Reply:
Multiple directories are more automate-able, but please figure out what would need to on `packages` (just a listing? something else?)

Reply:
Okay. That works. Do you think it would look funny if there is only one? E.g. a bulleted list that just has `1.14.0`?

Reply:
On one hand, I think it's okay to be very explicit/verbose on which release of the OS (1.14.0 instead of 1.14.x) a user is reading about

However on the other hand, not every release gets patch releases, so a lot of the time there may just be one item on the list as you said.

Would it be better if the initial default is to make `packages/` show the e.g. 1.14.0 release package list, then if there is a 1.14.1 released, 1.14.0 is placed in a subdirectory alongside 1.14.1?

Or: do we not want to move pages around after releasing them?

Reply:
Okay. And yeah, let's not create a situation that changes URLs.